srcpkgs/bmap-tools/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'bmap-tools'
+pkgname=bmap-tools
+version=3.5
+revision=1
+archs=noarch
+build_style=python3-module
+make_install_args="--optimize=1"
+hostmakedepends="python3-setuptools"
+depends="python3-setuptools python3-six"
```

Comment:
please add checkdepends="${depends}"

Reply:
I've made the change, though I'm curious as to what this does.

Reply:
Xbps-src can run test suite of packaged software. It's run with `check` target or when `XBPS_CHECK_PKGS=yes` is set in etc/conf. Without dependencies, tests failed.

Reply:
Thanks :+1: